lib/curator.js

```diff
@@ -141,9 +141,23 @@ bedrock.events.on('bedrock-express.configure.routes', function(app) {
     '/tasks/credentials/compose-identity',
     relaxedAuthenticate, acceptableContent('urlencoded'),
     parseForm, parseJsonData, function(req, res, next) {
+    var credentialQuery = req.body.query;
+
+    /*
+     * pubicKey has the following properties
+     * type: array that always contains 'CryptographicKey' and can also contain
+     * ‘EmphemeralCryptographicKey’ or ‘NewCryptographicKey’
+     * id:, publicKeyPem:
+     */
+    var publicKey = req.body.publicKey;
```

Comment:
We should add a `TODO` here that says that framing the public key may be necessary, it's TBD.